### DISCUSSION TITLE
Switch port

### DISCUSSION THREAD

**Discussion by Monkeor** [HarbourMasters/2ship2harkinian#501] [Ideas]
I played and still continue to play SoH exclusively on my Switch. Is there any chance to make 2S2H run on the Switch as well?

**Comment by hobbo91** [comment]
Please! I am in the exact same situation and someone compiling/porting this to the Switch would be awesome.

**Comment by hobbo91** [comment]
I agree! Please someone port to the Switch. I've never actually done a full play though of MM and the Switch would be the perfect platform for this.

**Comment by Callero666** [comment]
Waiting for someone with knowledge to do it too

**Comment by disavow-commits** [comment]
There is a working switch port now.  It is created by Yoshicrystal9.  I hope it gets merged in with these main 2s2h github files. https://github.com/YoshiCrystal9/2ship2harkinian-Switch
